docker/mysql_tests/scripts/base_tests/journals.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+set -e -x
+
+. /usr/local/export_common.sh
+
+export WALE_S3_PREFIX=s3://mysql_journal_test_bucket
+
+get_journal_count() {
+    wal-g st ls basebackups_005/ 2>&1 | grep journal_ | awk '{ printf $7 "\n" }' | wc -l
+}
+
+get_journal_name() {
+    wal-g st ls basebackups_005/ 2>&1 | grep journal_ | awk '{ printf $7 "\n"}' | sort | awk "FNR == $1 {print}"
+}
+
+get_backup_name() {
+    JOURNAL_NAME=$(get_journal_name $1)
+    echo ${JOURNAL_NAME#"journal_"}
+}
+
+get_journal_size() {
+    JOURNAL_NAME=$(get_journal_name $1)
+    wal-g st cat basebackups_005/$JOURNAL_NAME | jq '.JournalSize'
+}
+
+mysqld --initialize --init-file=/etc/mysql/init.sql
+service mysql start
+sysbench --table-size=10 prepare
+
+# Create backup #1 with journals
+sysbench --time=5 run
+mysql -e 'FLUSH LOGS'
+wal-g binlog-push
+wal-g backup-push --count-journals
+sleep 1
+
+# Check count of backups and content of them
+test "1" -eq $(get_journal_count)
+test "0" -eq $(get_journal_size 1)
+
+# Create backup #2 with journals
+sysbench --time=5 run
+mysql -e 'FLUSH LOGS'
```

Comment:
'FLUSH LOGS' works perfectly fine for your test case... so, let's reduce `--time=5` to smaller waits.